docs/src/userguide/language_basics.rst

```diff
@@ -657,10 +939,25 @@ direct equivalent in Python.
 * There is no unary ``*`` operator in Cython. Instead of ``*p``, use ``p[0]``
 * There is an ``&`` operator, with the same semantics as in C.
 * The null C pointer is called ``NULL``, not ``0`` (and ``NULL`` is a reserved word).
-* Type casts are written ``<type>value`` , for example,::
+* Type casts are written ``<type>value`` , for example,
+
+  .. tabs::
 
-        cdef char* p, float* q
-        p = <char*>q
+      .. group-tab:: Pure Python
+
+          .. code-block:: python
+
+              p: cython.p_char
+              q: cython.p_float
+              p = cython.cast(cython.p_char, q)
+
+      .. group-tab:: Cython
+
+          .. code-block:: cython
+
+              cdef char* p, float* q
+
+              p = <char*>q
```

Comment:
take a look [here](https://github.com/cython/cython/pull/4242/files#r660136709)

Reply:
Alternatively, you can split `cdef` across two lines.

Reply:
cdef splitted across two lines.